src/main/java/com/snowflake/kafka/connector/internal/SnowflakeURL.java

```diff
@@ -34,9 +34,27 @@
 
   private int port;
 
+  /**
+   * There are several matching groups here. Matching groups numbers are identified as the opening
+   * braces start and are indexed from number 1.
+   *
+   * <p>Group 1: If https is present or not. (Not required)
+   *
+   * <p>Group 2: Is the entire URL including the port number
+   *
+   * <p>Group 3: URL until .com
+   *
+   * <p>Group 4: Account name (may include org-account/alias)
+   *
+   * <p>Group 5: (Everything after accountname or org-accountname until .com)
+   *
+   * <p>Group 7: port number
+   */
+  private static final String SNOWFLAKE_URL_REGEX_PATTERN =
```

Comment:
Is this a copy+paste from snowflakedb repo? If so, can you include a link to that here as a comment?

Reply:
Nah, I added this comment for regex amateurs like me. It is to validate the url before generating one for jdbc.